[IMP] purchase: replace purchase uom screenshots (#9317)
odoo/documentation: content/applications/inventory_and_mrp/purchase/products/uom.rst

=========================
Purchase units of measure
=========================

When you purchase a product, it may happen that your vendor uses a different unit of measure than

> you do when you sell it. This can cause confusion between sales and purchase representatives. It is

Reviewer: ```suggestion
when it is sold This can cause confusion between sales and purchase representatives. It is
```
please ensure the rest of the doc removes any instance of "you" or "your" -- Thanks

Author: Changing this specific instance in this suggestion, but I am not making any other updates to the copy outside of it, per my note at the top of the PR:

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

Updates to the copy (which I know are needed) will follow in a separate PR once this is merged.

   :align: center
   :alt: Create a new units of measure category in Odoo Purchase

The next step is to create the two units of measures. To do so, click into the Unit of Measure
category. Under the :guilabel:`Units of Measure` tab, click :guilabel:`Add a line`.

First, create the unit of measure used as the reference point for converting to other units of

> measure inside the category. Name the unit and select the units of measure

Reviewer 2: Early line break (move category to line 77)

Author: Suggestion implemented ✅

> measure inside the category. Name the unit and select the units of measure

Reviewer 2: ```suggestion
measure inside the category. Name the unit, and select the units of measure
```

Author: Suggestion implemented ✅

   :align: center
   :alt: Create a new units of measure category in Odoo Purchase

> The next step is to create the two units of measures. To do so, click into the Unit of Measure
> category. Under the :guilabel:`Units of Measure` tab, click :guilabel:`Add a line`.

Reviewer: ```suggestion
The next step is to create the two units of measures. To do so, click into the :guilabel:`Unit of Measure
Category` field and enter a name for the category. Then, under the :guilabel:`Units of Measure` tab, click :guilabel:`Add a line`.
```

Author: Suggestion implemented ✅

First, create the unit of measure used as the reference point for converting to other units of
measure inside the category. Name the unit, and select the units of measure category you just
created. For the *Type*, select *Reference Unit of Measure for this category type*. Enter the
*Rounding Precision* you would like to use. The quantity computed by Odoo is always a multiple of

> this value.

Reviewer: remove the use of "you" and correct the incorrect usage of italics here, please

Author: I am not making any other updates to the copy outside of the one implemented suggestion, per my note at the top of the PR:

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

Updates to the copy (which I know are needed) will follow in a separate PR once this is merged.
